src/api/track.ts

```diff
@@ -1,3 +1,7 @@
+import { detectIncognito } from 'detectincognitojs';
+// @ts-ignore
+import { KJUR } from 'jsrsasign';
```

Comment:
I think we can just import `{ jws }` instead of `KJUR`, which is the GitHub author's name... https://github.com/kjur/jsrsasign/blob/master/npm/lib/jsrsasign.js#L341

Also, this is a pretty massive library to be including just for this use case. We should see if there's a more appropriate package, or just pull in the pieces we need (this code probably doesn't change very often).